compiler/x/codegen/OMRX86Instruction.cpp

```diff
@@ -991,7 +991,8 @@ void TR::X86RegRegInstruction::assignRegisters(TR_RegisterKinds kindsToBeAssigne
 
       bool regRegCopy = isRegRegMove();
       TR_X86OpCodes opCode = getOpCodeValue();
-      if (feGetEnv("TR_UseOutOfDatedRegRevMoveList"))
+      static char* env = feGetEnv("TR_UseOutOfDatedRegRevMoveList");
```

Comment:
I do take exception to a couple of spelling mistakes in the environment variable itself.  Shouldn't the spelling be `TR_UseOutOfDateRegRegMoveList` or am I completely missing the point of this env var?

Rather than `env`, why not call the auto `useOutOfDateRegRegMoveList` to make it explicit?

Reply:
I am not certain about the original point of the env var, but based on the context the spelling you propose seems valid.